backend_addon/{{ cookiecutter.__folder_name }}/pyproject.toml

```diff
@@ -1,15 +1,98 @@
-# Generated from:
-# https://github.com/plone/meta/tree/main/config/default
-# See the inline comments on how to expand/tweak this configuration file
+[project]
+name = "{{ cookiecutter.python_package_name }}"
+dynamic = ["version"]
+description = "{{ cookiecutter.description }}"
+readme = "README.md"
+license = "GPL-2.0-only"
+requires-python = ">={{ cookiecutter.__supported_versions_python[0] }}"
+authors = [
+    { name = "{{ cookiecutter.author }}", email = "{{ cookiecutter.email }}" },
+]
+keywords = [
+    "CMS",
+    "Plone",
+    "Python",
+]
+classifiers = [
+    "Development Status :: 3 - Alpha",
+    "Environment :: Web Environment",
+    "Framework :: Plone",
+    {%- for version in cookiecutter.__supported_versions_plone -%}
+    "Framework :: Plone :: {{ version }}",
+    {%- endfor %}
+    "Framework :: Plone :: Addon",
+    {%- if cookiecutter.__feature_distribution == '1' %}
+    "Framework :: Plone :: Distribution",
+    {%- endif %}
+    "License :: OSI Approved :: GNU General Public License v2 (GPLv2)",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python",
+    {%- for version in cookiecutter.__supported_versions_python -%}
+    "Programming Language :: Python :: {{ version }}",
+    {%- endfor %}
+]
+dependencies = [
+    "Products.CMFPlone",
+    "plone.api",
+    {% if cookiecutter.__feature_headless == '1' -%}
+    "plone.restapi",
+    "plone.volto",
+    {%- endif %}
+]
+
+[project.optional-dependencies]
+test = [
+    "plone.app.testing",
+    {% if cookiecutter.__feature_headless == "0" and cookiecutter.plone_version >= "6.1" -%}
+    "plone.classicui",
+    {%- endif -%}
+    "plone.restapi[test]",
+    "pytest",
+    "pytest-cov",
+    "pytest-plone>=0.5.0",
+]
+
+[project.urls]
+Homepage = "{{ cookiecutter.__repository_url }}"
+PyPI = "https://pypi.org/project/{{ cookiecutter.python_package_name }}"
+Source = "{{ cookiecutter.__repository_url }}"
+Tracker = "{{ cookiecutter.__repository_url }}/issues"
+
+{%- if cookiecutter.plone_version.startswith('6.1') is true %}
+[project.entry-points."plone.autoinclude.plugin"]
+{%- else %}
+[project.entry-points."z3c.autoinclude.plugin"]
+{%- endif %}
+target = "plone"
+
+[tool.uv]
+managed = {{ cookiecutter.__backend_managed_by_uv }}
+
+[tool.hatch.version]
+path = "src/{{ cookiecutter.__package_path }}/__init__.py"
+
 [build-system]
-requires = ["setuptools>=68.2,<=75.8.0"]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[tool.hatch.build]
+strict-naming = true
+
+[tool.hatch.build.targets.sdist]
+exclude = [
+  "/.github",
+]
+
+[tool.hatch.build.targets.wheel]
+packages = ["src/{{ cookiecutter.__package_path }}"]
```

Comment:
This is not working for namespace packages.

1. Generate a project named collective.something
2. cd to collective-something/backend and run `make test`

```
ImportError while loading conftest '/Users/davisagli/Plone/collective-something/backend/tests/conftest.py'.
tests/conftest.py:1: in <module>
    from collective.something.testing import ACCEPTANCE_TESTING
E   ModuleNotFoundError: No module named 'collective.something'
make: *** [test] Error 4
```

Reply:
Fixed and added a simple GHA test to make sure things work